buildconfig/pygame-stubs/__init__.pyi

```diff
@@ -1,71 +1,55 @@
-from typing import Any, Tuple, Callable, Union, Optional, overload, Type
+from typing import Callable, Optional, Tuple, Type, Union
 
-# Most useful stuff
-from pygame.constants import *
-import pygame.surface
-import pygame.rect
-import pygame.color
-import pygame.event
-import pygame.bufferproxy
-import pygame.draw
-import pygame.display
-import pygame.font
-import pygame.image
-import pygame.key
-import pygame.mixer
-import pygame.mouse
-import pygame.time
-import pygame.version
-
-# Advanced stuff
-import pygame.cursors
-import pygame.joystick
-import pygame.mask
-import pygame.sprite
-import pygame.transform
-import pygame.bufferproxy
-import pygame.pixelarray
-import pygame.pixelcopy
-import pygame.sndarray
-import pygame.surfarray
-import pygame.math
-import pygame.fastevent
-
-# Other
-import pygame.scrap
+from . import bufferproxy as bufferproxy
```

Comment:
I'm not sure if this syntax is ok with py2/mypy/jedi/etc?

Reply:
Re-read the specs and recommendations on this. Will update

Reply:
fixed
Changed all the "from . import foo as foo" to " from . import foo"